compose

```diff
@@ -6,7 +6,7 @@ set -o errexit
 if [[ -f '.compose.env' ]]; then
   set -o allexport
   # avoid override already set variables
-  source <(grep -v '^#' .compose.env | sed -E 's|^(.+)=(.*)$|: ${\1=\2}; export \1|g')
+  source .compose.env
```

Comment:
this is fixed by https://github.com/ansible/galaxy_ng/pull/689

Reply:
I accidentally committed that. Oops haha